frontend/src/assets/svgs/settings.svg

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg width="36px" height="36px" viewBox="0 0 36 36" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg width="200px" height="200px" viewBox="0 0 36 36" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
You don't need to change the svg dimension. One webpack plugin will automatically strip those dimensions away from svg component. CSS modification should be sufficient.

Reply:
I realized that as I was testing it and added the CSS modification to SettingsButton.css